src/App/FirstTask.java

```diff
@@ -0,0 +1,7 @@
+package App;
+
+public class FirstTask {
+    public boolean booleanExpression(boolean a,boolean b,boolean c,boolean d){
+        return ((a ^ b) && (c ^ d)) || ((a ^ c) && (b ^ d)) || ((a ^ d) && (b ^ c));
```

Comment:
you can make it more simply

Reply:
Ctrl+Alt+L

Reply:
return ((a ^ b) && (c ^ d)) || ((a ^ c) && (b ^ d)) || ((a ^ d) && (b ^ c)); 
->
return ((a ^ b) && (c ^ d)) || ((a ^ c) && (b ^ d));

Reply:
the same mistake with PR :( https://github.com/AristarkhV/Homework/pull/1/commits/131a80ae7fc2976bfd69454c429bc942135486da